apps/test/unit/templates/sectionProgressV2/SectionProgressSelectorTest.jsx

```diff
@@ -141,8 +141,6 @@ describe('SectionProgressSelector', () => {
   it('sets user preference when link clicked', () => {
     renderDefault();
 
-    const remindLaterLink = screen.getByText('Remind me later');
```

Comment:
Why are we removing this?

Reply:
In the original tests we added this in because the pop-up got in the way of viewing the screen with the SectionProgressSelector in it.  With the changes to how the pop-up is rendered, it will not render the pop-up until it has data back from Redux about if the pop-up `hasBeeenSeen` or if there is a `dateLastDelayed`.  We could make add these reducers to this test, but given that this test is more about testing the SectionProgressSelector and we already have eyes tests that ensure that this works, I was thinking that wasn't necessary.  Thoughts?